api/src/main/java/org/pmiops/workbench/db/dao/UserService.java

```diff
@@ -116,18 +116,28 @@ private void updateDataAccessLevel(User user) {
       !configProvider.get().access.enableEraCommons || user.getEraCommonsCompletionTime() != null;
     boolean complianceTrainingCompliant = user.getComplianceTrainingCompletionTime() != null ||
       user.getComplianceTrainingBypassTime() != null || !configProvider.get().access.enableComplianceTraining;
-    boolean idVerificationCompliant = user.getIdVerificationCompletionTime() != null ||
-      user.getIdVerificationBypassTime() != null || !configProvider.get().access.enableIdVerification ||
-    // TODO: can be removed once we totally move off old validation
-      Optional.ofNullable(user.getIdVerificationIsValid()).orElse(false);
+    boolean betaAccessGranted = user.getBetaAccessBypassTime() != null ||
+            !configProvider.get().access.enableBetaAccess;
+
+    // TODO: Add in when we add idVerification module
```

Comment:
IMO we should probably just clear out these TODOs. I've heard reasonable people argue that commented-out code should categorically never be left in place. I try not to be too much of a stickler there, but at this point I think the patterns are well-set enough that we don't need to keep this around.